lib/public/components/pickers/tag/PickerModel.js

```diff
@@ -16,22 +16,23 @@ import { Observable } from '/js/src/index.js';
 /**
  * Model to handle the state of a tag picker
  */
-export class TagPickerModel extends Observable {
+export class PickerModel extends Observable {
     /**
      * Model's constructor, collapsed and without selection as a default (default are used as initial state and when the picker is reset)
      *
+     * @param {string} name The name of the
      * @param {Object[]} defaultSelection the default selection of tags
      * @param {boolean} defaultCollapsed true if the tags must be collapsed as a default
-     *
      * @constructor
      */
-    constructor(defaultSelection = [], defaultCollapsed = true) {
+    constructor(name = '', defaultSelection = [], defaultCollapsed = true) {
```

Comment:
If you use the name only on the picker view I think it should be a parameter of this component